packages/calcite-components/src/components/stepper/stepper.stories.ts

```diff
@@ -212,3 +212,9 @@ export const verticalLayout_TestOnly = (): string => html`<calcite-stepper layou
       >Step 1 Content Goes Here</calcite-stepper-item
     >
   </calcite-stepper>`;
+
+export const firstItemDisabled_TestOnly = (): string => html`<calcite-stepper>
```

Comment:
Do we need this test if the E2E test already covers it?

Reply:
Since there are visual differences for selected stepper-item it is better to have a screenshot. I think we can utilize existing `disabled` story. Thoughts?